### DISCUSSION TITLE
Frontend publishing and registration for users

### DISCUSSION THREAD

**Discussion by Grosso2020** [GetPublii/Publii#1841] [Ideas]
I may have a crazy idea fundamentally breaking the CMS philosophy, but how about allowing users to publish posts? I understand the block editor is based on editor.js? It's good and would be fine for publishing to the site. If you would make such a plugin for CMS, I would buy it with pleasure and other users would too.
So my suggestion:
1. User authorization and registration
2. Form to publish post from user profile.
3. Moderation of posts and users.

**Comment by candidexmedia** [comment]
Honestly, I think something like Ghost would be better suited for those kinds of features. 

Maybe someone out there could work on that plugin, but I don't think Publii is meant for complex web publishing systems.

See related post: https://github.com/GetPublii/Publii/discussions/1478#discussioncomment-9125091
